### ISSUE TITLE
Accessibility issue: In DetailsList, splitter is not accessible by keyboard while it can be accessed using mouse

### ISSUE THREAD

**Issue by ajinkya599** [OfficeDev/office-ui-fabric-react#5239]
#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Products/sites affected: vsts pages which are using DetailsList - Library, Task Groups, Deployment Groups, Work Items etc.

#### Describe the issue:

Accessibility Issue:
DetailsList- Splitter is not accessible by keyboard 

There are multiple pages in vsts, which are using DetailsList to render a list.
The splitter (used to resize the column width) is accessible using the mouse, but cannot be accessed using the keyboard)

#### Repro:
For any project on a visual studio account (eg. office.visualstudio.com), navigate to any of the following pages (The page should have a populated list):
Library, Task Groups, Deployment Groups, Work Items etc.

Repro URLs:
https://office.visualstudio.com/APEX/_library?itemType=VariableGroups
https://office.visualstudio.com/APEX/_taskgroups

#### Actual behavior:

While navigating, splitter in the column header is not accessible by using keyboard, whereas it is accessible by mouse.

#### Expected behavior:

While navigating, splitter in the column header should be accessible by using keyboard.

**Comment by betrue-final-final** [comment]
Legit issue. I saw at an assistive technology conference where they used slider functionality to make this happen. @lynamemi, we should talk about this one!

**Comment by micahgodbolt** [comment]
duplicate of #4947
